### ISSUE TITLE
Comments for Liquid Galaxy becomes VisionPort

### ISSUE THREAD

**Issue by sethjensen1** [EndPointCorp/end-point-blog#1800]
Comments for https://www.endpointdev.com/blog/2021/11/liquid-galaxy-becomes-visionport/
By Alejandro Ramon

To enter a comment:

1. Log in to GitHub
2. Leave a comment on this issue.

**Comment by jandro-air** [comment]
Yes, the product name moving forward has now officially been changed to VisionPort. Liquid Galaxy continues to exist as a technology, but our platform is VisionPort. Thank you for rolling with the changes so well!

We have a few sites to add to our installations page now, but I will make sure Auburn is prioritized :smiley:

**Comment by sethjensen1** [comment]
@chmixon Auburn University is now on proud display on our Installations page!
